packages/client/src/comp/Blurb.tsx

```diff
@@ -1,141 +1,75 @@
 import React from 'react'
 import styled from 'styled-components'
-import { toast } from 'react-toastify'
-import { RoundedButton } from './util/Button'
-import { StyleContainer } from './util/Container'
-import { useAppHistory } from '../lib/path'
-import { client } from '../lib/trpc'
-import { AddressContainer, FetchingDataContainer } from '../lib/unstated'
-import { AppForm } from './util/Form'
+import { Container } from 'muicss/react'
+import { cssQuery } from './util/cssQuery'
 
+import img1k from './img/blurb_bg_max_width_1k.jpg'
+import img2k from './img/blurb_bg_max_width_2k.jpg'
+import img6k from './img/blurb_bg_max_width_6k.jpg'
+import { EnterZip } from './util/EnterZip'
+import { MarketingWrapper } from './util/MarketingWrapper'
 
-const Background = styled.div`
-  top: 0;
-  left: 0;
-  min-width: 100%;
-  background: rgb(144,202,249);
-  background: linear-gradient(-45deg, rgba(144,202,249,1) 0%, rgba(30,136,229,1) 25%, rgba(21,101,192,1) 100%);
-  color: #f1f1ff;
-`
-
-const Title = styled.h1`
-  font-weight: 100;
-  padding-top: 24px;
-  @media only screen and (max-width: 544px) {
-    padding-top: 8px;
-  }
-`
-
-const Text = styled.p`
-  margin-bottom: 16px;
-`
-
-const FlexBox = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: flex-start;
-`
-
-const FlexContainer = styled.div`
-  margin-top: 24px;
-  display: flex;
-  flex-direction: row;
-  justify-contnet: flex-start;
-  align-content: center;
-`
+const bgGradient = 'linear-gradient(to bottom right, #fffd, #fffa)'
 
-const ZipInput = styled.input`
-  padding: 14px 16px;
-  border: none;
-  box-shadow: none;
-  outline: none;
-  width: 84px;
-  border-radius: 4px;
-  margin-right: 1rem;
-`
+const Wrapper = styled(MarketingWrapper)`
+  position: relative;
+  min-height: 100vh;
+  z-index: 9;
+  box-shadow: 0 0 14px #0003;
 
-const SubmitButton = styled(RoundedButton)`
-  z-index: 0;
-  background: #4DB6AC;
-  color: #f1f1ff;
-  margin 0;
-  :hover {
-    background: #5DC6BC;
-    color: #f1f1ff;
+  /*
+    Loads the background image according to the device's screen dimensions
+    We use manual queries here since these are mostly based on the dimensions
+    of the pictures used as background.
+  */
+  @media screen and (max-width: 640px) {
+    background-image: ${bgGradient}, url(${img1k});
+    background-size: cover;
+    background-position-x: 64%;
+    background-position-y: 0;
+  }
+  @media screen and (min-width: 641px) {
```

Comment:
Why don't you use `cssQuery.ts`?

Reply:
I'm not using `cssQuery` here because these queries are relative to the images' sizes, for example we don't have a `cssQuery.veryWide` for desktops/tablets above `1920px` but we do have these images in dimensions bigger than this width. We could simplify the query but [there's a great amount of screens width available between 1280 and 1920](https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide) (1440, 1600, 1536, etc.) so a lot of users could benefit from more specific queries.
Since the image rendered is always from a file that is a bit wider than users' screens, using a query for large devices (above 1200) would result in most users downloading images that are far bigger than what they actually need.